driver-core/src/main/com/mongodb/client/model/GridFSDownloadByNameOptions.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model;
+
+/**
+ * The GridFS download by name options
+ *
+ * <p>Controls the selection of the revision to download</p>
+ *
+ * @since 3.1
+ */
+public class GridFSDownloadByNameOptions {
```

Comment:
Move to com.mongodb.client.gridfs.model.  I'd rather keep gridfs as separate as possible short of introducing new artifacts.